.env.example

```diff
@@ -12,4 +12,6 @@ SHOW_ACTION_MESSAGE=true
 SHOW_JOIN_MESSAGE=true
 SHOW_KICK_MESSAGE=true
 SHOW_LEAVE_MESSAGE=true
+USE_IMGUR_FOR_IMAGES=false
+IMGUR_CLIENT_ID=0000000000
```

Comment:
Also, while the environment variables are nice if you're using Docker, we should also have a way for people to use the configuration file, `config.js`, to configure the Imgur support. Can you make sure that this functionality is included into `config.js.example`?